english/light-fire/run-away/follow/follow.md

```diff
@@ -1,4 +1,13 @@
 You try to run after the child, noticing that it is covered in melted marshmallows.
 You realize that you also are covered in melted marshmallow goo that is quickly cooling.
 Each step you take gets more and more difficult until all of the goo cools and hardens.
-You are now a tasty statue, nibbled at by mice and birds.
\ No newline at end of file
+You are now a tasty statue, nibbled at by mice and birds.
+
+Hours have passed and it is now night time. The mice and birds have nibbled you out of you statue. You rejoice in silent glee that you've made it to the outside, but you get the instant feeling of dread.
+Standing in the field, you start to hear shrill screams coming from your apartment. You start, first, at a brisk walk. Then, as you get closer, and the screaming is louder you run towards the noise. That voice is familiar. It's your best friend!
+
+What do you do?
+
+[turn away from the volcanic marshmallow glob that was your apartment and run?](../friend-scream/walk-away.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.